app/models/caches/current_hbx.rb

```diff
@@ -6,7 +6,7 @@ def self.fetch
     end
 
     def self.cache!
-      Thread.current[:__current_hbx_lookup_cache] = HbxProfile.find_by_state_abbreviation(HbxProfile.aca_state_abbreviation)
+      Thread.current[:__current_hbx_lookup_cache] = HbxProfile.find_current_with_proxy_inbox
```

Comment:
@TreyE Please update the dummy classes(HbxProfile & CurrentHbx) in the Financial Assistance Engine.